src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -72,16 +72,17 @@ const ContractInteractionReview = ({ onClose, onPrev, tx }: Props): React.ReactE
     const txRecipient = tx.contractAddress
     const txData = tx.data ? tx.data.trim() : ''
     const txValue = tx.value ? toTokenUnit(tx.value, nativeCoin.decimals) : '0'
-    dispatch(
-      createTransaction({
-        safeAddress: safeAddress as string,
-        to: txRecipient as string,
-        valueInWei: txValue,
-        txData,
-        notifiedTransaction: TX_NOTIFICATION_TYPES.STANDARD_TX,
-      }),
-    )
-
+    if (safeAddress) {
+      dispatch(
+        createTransaction({
+          safeAddress,
+          to: txRecipient as string,
+          valueInWei: txValue,
+          txData,
+          notifiedTransaction: TX_NOTIFICATION_TYPES.STANDARD_TX,
+        }),
+      )
+    }
     onClose()
```

Comment:
are we going to close the modal and won't give any feedback to the user? I don't think this is designed but feels a bit odd.

Probably we should somehow inform the user about the issue or at least, log an error. WDYT?

Reply:
:point_up_2: Same message for all the other changes

Reply:
We currently don't have any way to display errors as modals to the user, maybe we can display a log on the console and show a notification? what do you think?

Reply:
I added console errors for those cases